views/application.jade

```diff
@@ -72,6 +72,9 @@ block content
                             .form-group
                                 label(for='appname') Application Name:
                                 input(value=application.name name='appname').form-control
+                            .form-group
+                                label(for='appdesc') Application Description:
+                                input(value=application.description name='appdesc').form-control
```

Comment:
I would rather have a div here with Markdown enabled content; `marked` should be present at least on the backend, so using that here should be really simple. Use `!{someVariable}` in the Jade code to not HTML encode the content, but take the things as is.

Is there a reason why we should not enable Jade here?